spec/spec_helper.rb

```diff
@@ -4,6 +4,7 @@
 require 'rubygems'
 require 'rspec'
 require 'rspec/autorun'
+require 'rspec/its'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.